msteams-platform/concepts/build-and-test/analyze-your-apps-usage-in-developer-portal.md

```diff
@@ -46,57 +49,189 @@ After you have selected your desired filters, you can explore the following indi
+   | **Median days used per month** | The median number of days in which your app was opened in the last R30 (Rolling 30 days) time period. |
+   | **% of 5+ Days usage** | The percentage of active users who opened or used the app more than five days in the last R30 time period. |
+   | **DAU/MAU** | The ratio of the average number of unique users or tenants who used your app on each day divided by the Monthly active users for the selected R30 time period. |
+
+ #### My app dashboard
+
+   The **My App Dashboard** table shows you the latest R30 (Rolling 30 days) data for each of the metrics under the previous four categories, and the Month over Month change. Use the time picker on the upper right side and select the desired date to view the following:
+
+* Daily R30 data for the last 75 days.
+* End of month R30 data for up to 12 months.
+
+   You can select each of these metric names to see trends over time.
+
+   :::image type="content" source="../../assets/images/tdp/tdp-app-dashboard-view.png" alt-text="Screenshots shows you app capability usage of your published app in Developer Portal."lightbox="../../assets/images/tdp/tdp-app-dashboard-view.png":::
+
+ ### Host product
+
+   The host product page shows you the following app usage analytics:
+
+   * [Usage by host product](#usage-by-host-product)
+   * [Usage by operating system](#usage-by-operating-system)
+   * [Usage by platform](#usage-by-platform)
+
+   The Key Performance Indicator (KPI) cards display metrics such as the host with the most active users and the operating system (OS) with the most active users, based on monthly, weekly, and daily data, in a metric card format.
+
+   :::image type="content" source="../../assets/images/tdp/tdp-host-product.png" alt-text="Screenshots shows you app usage in different host products of your published app in Developer Portal.":::
+
+ #### Usage by host product
+
+   The **Usage by host product** chart shows your app's active usage across various host products, including as **Teams**, **Outlook**, and the **Microsoft 365 app** of monthly, weekly, and daily data.
+
+   :::image type="content" source="../../assets/images/tdp/tdp-host-usage.png" alt-text="Screenshots shows you app usage in different host product of your published app in Developer Portal.":::
+
+ #### Usage by platform
+
+   The **Usage by platform** chart shows your app's active usage across various platforms, such as **Desktop**, **Mobile**, and **Web** of monthly, weekly, and daily data.
+
+   :::image type="content" source="../../assets/images/tdp/tdp-usage-platform.png" alt-text="Screenshots shows you app usage in different platform of your published app in Developer Portal.":::
+
+   > [!NOTE]
+   > The `Preview` tag indicates that the metric **Usage by host product** is in its initial phase and might undergo modifications as we collect feedback and continue to improve the experience.
```

Comment:
Is there a place we explain the known data issues? For example, these are a list of known issues we discussed earlier. Please check if any of them should be surfaced to users: 

- Usage data from Outlook ME and Outlook Add-in is not yet included.
- The Outlook personal tab currently includes both personal tab and tab (meeting scenario) data.
- Consumer apps usage is not included.
- Usage data from Outlook Android that lacks tenant ID is not included. 
- Some usage events have unknown element types due to known telemetry issues in Teams; fixes are currently in progress.

Reply:
@srinidyMSFT , could you please confirm if we can add the above limitations.

Reply:
@v-sdhakshina - we should have these limitations mentioned part of the documentation. however @surbhigupta12 could you please let me know what is the common guidance/practice around the same?

Reply:
Updated as known limitations.